sift/client.py

```diff
@@ -313,6 +314,135 @@ def get_workflow_status(self, run_id, timeout=None):
         except requests.exceptions.RequestException as e:
             raise ApiException(str(e))
 
+    def get_decisions(self, entity_type, limit, start_from, abuse_types, timeout=None):
+        """Get decisions available to customer
+
+        Args:
+            entity_type: only return decisions applicable to entity type {USER|ORDER}
+            limit: number of query results (decisions) to return [default: 100]
+            start_from: result set offset for use in pagination [default: 0]
+            abuse_types: csv of abuse_types to filter returned decisions by (optional)
+
+        Returns:
+            A sift.client.Response object containing array of decisions if call succeeded
+            Otherwise raises an exception
+        """
+
+        if timeout is None:
+            timeout = self.timeout
+
+        params = {}
+
+        if entity_type:
+            params['entity_type'] = entity_type
+        if limit:
+            params['limit'] = limit
+        if start_from:
+            params['from'] = start_from
+        if abuse_types:
+            params['abuse_types'] = abuse_types
+
+        try:
+            return Response(requests.get(self._get_decisions_url(self.account_id), params=params,
+                                         auth=requests.auth.HTTPBasicAuth(self.api_key, ''),
+                                         headers={'User-Agent': self._user_agent()}, timeout=timeout))
+
+        except requests.exceptions.RequestException as e:
+            raise ApiException(str(e))
+
+    def apply_user_decision(self, user_id, apply_decision_request, timeout=None):
+        """Apply decision to user
+
+        Args:
+            user_id: id of user
+            applyDecisionJson:
+                decision_id: decision to apply to user
+                source: {one of MANUAL_REVIEW | AUTOMATED_RULE | CHARGEBACK}
+                analyst: id or email, required if `source: MANUAL_REVIEW`
```

Comment:
For `analyst` field, we are not limiting clients to Sift analyst emails. We will accept any id string.